operations/mimir-mixin/config.libsonnet

```
@@ -30,7 +30,7 @@
       ruler_query_frontend: '(ruler-query-frontend.*)',  // Match also custom ruler-query-frontend deployments.
       query_scheduler: 'query-scheduler.*',  // Not part of single-binary. Match also custom query-scheduler deployments.
       ruler_query_scheduler: 'ruler-query-scheduler.*',  // Not part of single-binary. Match also custom query-scheduler deployments.
-      ring_members: ['alertmanager', 'compactor', 'distributor', 'ingester.*', 'querier.*', 'ruler', 'store-gateway.*', 'cortex', 'mimir'],
+      ring_members: ['alertmanager', 'compactor', 'distributor', 'ingester.*', '(ruler-)?querier.*', 'ruler', 'store-gateway.*', 'cortex', 'mimir'],
```

Let's keep the regexes stupid-simple.

```suggestion
      ring_members: ['alertmanager', 'compactor', 'distributor', 'ingester.*', 'querier', 'ruler', 'ruler-querier', 'store-gateway.*', 'cortex', 'mimir'],
```

(Why did we use `querier.*` before? We don't use such job names.)

---

I could trace it to https://github.com/grafana/cortex-jsonnet/pull/301 and https://github.com/grafana/cortex-jsonnet/pull/301

seems like it is so that we can have custom deployments of the queriers? Not really sure what that means. Do you remember?

I'll accept the suggestion to simplify the regex, should I also drop the `.*`?

---

```suggestion
      ring_members: ['alertmanager', 'compactor', 'distributor', 'ingester.*', 'querier.*', 'ruler', 'ruler-querier.*', 'store-gateway.*', 'cortex', 'mimir'],
```

---

Personally I would drop `.*` from querier, as we don't use it in our jsonnet. If anyone has special needs, they can extend `ring_members` easily. (In that case, let's mention it in changelog). I you think we should keep it, that's fine with me too.

---

I'll drop it and mention it in the changelog

---

fair enough